src/Symfony/Bundle/FrameworkBundle/Test/KernelTestCase.php

```diff
@@ -208,6 +208,8 @@ protected static function createKernel(array $options = [])
     }
 
     /**
+     * @after
+     *
      * Shuts the kernel down if it was used in the test.
      */
     protected static function ensureKernelShutdown()
```

Comment:
could you please confirm it works even if this method is protected?

Reply:
Can confirm it was called in a simple test extending the KernelTestCase.